keps/sig-node/4580-deprecate-kubelet-runonce/kep.yaml

```
@@ -0,0 +1,42 @@
+title: Deprecate & remove Kubelet RunOnce mode
+kep-number: 4580
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-node
+participating-sigs:
+status: provisional
+creation-date: 2024-04-17
+reviewers:
+  - "@tallclair"
+  - "@dchen1107"
+approvers:
+  - TBD
+
+see-also:
+
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.31"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.31"
+  beta: "v1.32"
+  stable: "v1.34"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: RemoveRunonce
```

Name the gate after the feature, not the change you intend to make:
```suggestion
  - name: KubeletRunOnceMode
```